.github/workflows/update-priority.yml

```diff
@@ -0,0 +1,63 @@
+# Updates the Priority field on a project board when a [Pri] label gets added to an issue.
+name: Update Priority Field
+
+# Triggers the workflow when a label is added to an issue.
+on:
+  issues:
+    types:
+      - labeled
+
+# Details needed to update a project field.
+env:
+  LABEL_PREFIX: "[Pri] "
+  PROJECT_NUMBER: 391
+  PROJECT_FIELD: "Priority"
+  ORGANIZATION: "Automattic"
+
+jobs:
+
+  # Updates the Priority field with the appropriate priority level.
+  update_priority: 
+
+    runs-on: ubuntu-latest
+
+    # Only runs if a priority label is added.
+    if:  contains(github.event.label.name, '[Pri]')
+
+    steps:
+    
+      # Generate a token for the GitHub App, which has special permissions to update projects.
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PEM }} 
```

Comment:
The standard GitHub token does not have the scope needed to access projects, so a new token has to be created. It was recommended [in this doc](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/automating-projects-using-actions#github-actions-workflows) to create a GitHub App for organization projects rather than a personal access token. I created one easily in my own repo, but I don't have the permission to do so in the Automattic organization.

Is that a permission I can get or is it not recommended? Is there already a token out there than has read and write permissions for organization projects, because that's all I need

Reply:
Oh nice! Great catch 👏 Based on the post, it sounds exactly like what I'm looking for! I'll wait for Jeremy to chime in though.

Reply:
Yup, that's what it's for! :)